src/experimental/reconciler.ts

```diff
@@ -28,15 +52,22 @@ const hostConfig = {
 		// trigger an immediate render to ensure <Static> children are written to output before they get erased
 		if (rootNode.isStaticDirty) {
 			rootNode.isStaticDirty = false;
-			rootNode.onImmediateRender();
+			if (rootNode.onImmediateRender) {
```

Comment:
```suggestion
			if (typeof rootNode.onImmediateRender === 'function') {
```

Reply:
Done in https://github.com/vadimdemedes/ink/pull/264/commits/14805dd8ee591c4fb772877e2a05d7219ea844ee